docs/userguide.md

```diff
@@ -59,7 +59,7 @@ Please refer to [Kubefed Concepts](./concepts.md) first before you go through th
 
 ## Prerequisites
 
-The kubefed deployment requires kubernetes version >= 1.11. The following
+The kubefed deployment requires kubernetes version >= 1.13. The following
```

Comment:
Should this be updated?  Fine to use a newer version, but we want to support older versions of kube if possible.

cc: @pmorie

Reply:
Thanks, @marun , I saw there is some updating for these docs, kubernetes version is updated to  >= 1.13, I'll merge the new updating, and update this PR.